metricbeat/module/prometheus/docker-compose.yml

```
@@ -9,3 +9,12 @@ services:
         PROMETHEUS_VERSION: ${PROMETHEUS_VERSION:-2.6.0}
     ports:
       - 9090
+  prometheus-host-network:
+    image: docker.elastic.co/integrations-ci/beats-prometheus:${PROMETHEUS_VERSION:-2.6.0}-1
+    build:
+      context: ./_meta
+      args:
+        PROMETHEUS_VERSION: ${PROMETHEUS_VERSION:-2.6.0}
+    ports:
+      - 9090
+    network_mode: host
```

Please extend `prometheus` service to reduce duplication.
```suggestion
  prometheus-host-network:
    extends:
      service: prometheus
    network_mode: host
```